packages/vx-shape/src/util/getBandwidth.ts

```diff
@@ -0,0 +1,9 @@
+import { AnyD3Scale } from '@vx/scale';
+
+export default function getBandwidth(scale: AnyD3Scale) {
+  const range = scale.range();
+  const domain = scale.domain();
+  return 'bandwidth' in scale
```

Comment:
now that this isn't inline where it was maybe trying to be succinct, I wonder if we could avoid the `range/domain` call if `'bandwidth' in scale` is `true`?

Reply:
fixed